tests/reduction/reduction_get_lambda.h

```diff
@@ -0,0 +1,267 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides common code for interaction with lambdas
+//
+*******************************************************************************/
+
+#ifndef __SYCL_CTS_TEST_REDUCTION_GET_LAMBDA_H
+#define __SYCL_CTS_TEST_REDUCTION_GET_LAMBDA_H
+
+#include "reduction_common.h"
+#include <functional>
+
+namespace reduction_get_lambda {
+
+/** @brief Construct lambda for interacting with reducer while
+ *         sycl::handler.parallel_for used sycl::range
+ *  @tparam VariableT Variable type from type coverage
+ *  @tparam FunctorT The type of the functor with which the test runs
+ *  @tparam UseCombineFlagT std::integral_constant type that let switch between
+ *          calling .combine() function or operator +, *, ^= e.t.c.
```

Comment:
```suggestion
 *          calling .combine() function or operator +, *, ^=, etc.

Reply:
I've applied this comment in this commit: 93364e17c64aebf054d2c7f722800ac0892bdc23